content-src/experiments/containers.yaml

```diff
@@ -0,0 +1,69 @@
+dev: true
+id: 12
+title: 'Containers'
+slug: containers
+locales:
+  - 'en'
+launch_date: '2017-02-21T22:00:00.00Z'
+incompatible:
+  'blok@mozilla.org': 'Tracking Protection'
+thumbnail: /static/images/experiments/containers/icon/thumbnail.png
+description: >
+  Contain yourselves! Create different containers for each of your online lives -  
+  your work self, your social self, your personal self - to stay organized and maintain privacy.
+introduction: >
+  Containers let you create profiles in Firefox for all of your online lives. Custom 
+  labels and color-coded tabs help keep different activities  - like online shopping, 
+  travel planning, or checking work email -  separate.  And each container stores cookies 
+  separately, so online trackers can’t connect your browsing in one container to another.  
+  You can keep your shopping self separate from your social self from your work self, 
+  without worrying about being followed around the Web. 
```

Comment:
Can we add a sentence about staying logged into the same site with multiple different accounts at the same time?